cmd/init.go

```diff
@@ -15,18 +15,32 @@ func init() {
 	root.AddCommand(initCmd)
 	initCmd.Flags().BoolP("confirm", "c", false, "Prompt to confirm all configuration options - $FAAS_CONFIRM")
 	initCmd.Flags().StringP("path", "p", cwd(), "Path to the new project directory - $FAAS_PATH")
-	initCmd.Flags().StringP("runtime", "l", faas.DefaultRuntime, "Function runtime language/framework. - $FAAS_RUNTIME")
+	initCmd.Flags().StringP("runtime", "l", faas.DefaultRuntime, "Function runtime language/framework. Default available runtimes: 'node', 'quarkus' and 'go'. - $FAAS_RUNTIME")
 	initCmd.Flags().StringP("templates", "", filepath.Join(configPath(), "templates"), "Extensible templates path. - $FAAS_TEMPLATES")
-	initCmd.Flags().StringP("trigger", "t", faas.DefaultTrigger, "Function trigger (ex: 'http','events') - $FAAS_TRIGGER")
+	initCmd.Flags().StringP("trigger", "t", faas.DefaultTrigger, "Function trigger. Available triggers: 'http' and 'events' - $FAAS_TRIGGER")
 
 	if err := initCmd.RegisterFlagCompletionFunc("runtime", CompleteRuntimeList); err != nil {
 		fmt.Println("Error while calling RegisterFlagCompletionFunc: ", err)
 	}
 }
 
 var initCmd = &cobra.Command{
-	Use:        "init <name>",
-	Short:      "Initialize a new Function project",
+	Use:   "init <name>",
+	Short: "Initialize a new Function project",
+	Long: `Initializes a new Function project
+
+Creates a new Function project at 'path'. If 'path' is unspecified, assumes the
```

Comment:
same remark on 'path' as mentioned in `create`. And this description is missing the part on the function name inherited from the leaf directory.